File: src/main/java/bwdm/informationStore/ReturnNode.java
package bwdm.informationStore;

public class ReturnNode extends Node{
	private String returnStr;

	ReturnNode(String _returnStr, int _nodeLevel) {
		returnStr = _returnStr;
		nodeLevel = _nodeLevel;
		isIfNode = false;
		ID = staticID++;
	}

	@Override
	public String getConditionOrReturnStr() {
		return returnStr;
	}
}

File: src/main/kotlin/bwdm/informationStore/ReturnNode.kt
package bwdm.informationStore

class ReturnNode internal constructor(returnStr: String, _nodeLevel: Int) : Node() {
    override val conditionOrReturnStr = returnStr
    override var isIfNode = false

    init {
        nodeLevel = _nodeLevel
        ID = Node.staticID++
    }
}
